docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst

```diff
@@ -1,54 +1,140 @@
 .. _intervention_crm_mgmt_outreach:
 
-Vivarium Modeling Strategy - Outreach Intervention 
-**************************************************
+=====================
+Outreach Intervention
+=====================
 
-Overview
-++++++++
+.. contents::
+   :local:
+   :depth: 1
 
-Adherence to SBP and LDL-c lowering therapies would increase through patient outreach. Methods for outreach include 
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - SBP
+    - Systolic Blood Pressure
+    - 
+  * - LDL-C
+    - Low-Density Lipoprotein Cholesterol
+    - 
+  * - CVD
+    - Cardiovascular Disease 
+    - 
+
+
+Intervention Overview
+---------------------
+
+Primary adherence to SBP and LDL-C lowering therapies would increase through patient outreach. Methods for outreach include 
 regular phone calls between provider and patient, a mobile app with reminders and guidance, or a patient support clinic.
 
 Today, few patients receive this support and adherence to medications is often low. This intervention will assess the 
 impact of an outreach intervention on risk factors and CVD. 
 
 
+.. list-table:: Affected Outcome #1 Effect Size
+  :widths: 15 15 15 
+  :header-rows: 1
+
+  * - OR for Primary Adherence
+    - Source 
+    - Notes
+  * - 2.16 
+    - [Derose-2013]_ 
+    - 
+
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+Assume there is no coverage today. 
+
+
+Vivarium Modeling Strategy
+--------------------------
+
 Eligibility and Initiation
 ++++++++++++++++++++++++++
 
-- SBP >=130 mmHg and/or LDL-c >= 2.6 mmol/L 
-- Must have an acute qualifying event (acute myocardial infarction or acute ishemic stroke) 
+- SBP >=130 mmHg and/or LDL-c >= 1.8 mmol/L 
 - Enrollment in the intervention only happens during interactions with healthcare, as shown :ref:`here <us_cvd_concept_model>`
 
 
 Affected Outcomes
 +++++++++++++++++
 
-This intervention affects adherence to medications, which in turn will affect the simulants exposure to SBP and LDL-c. 
+.. note::
+  There is data by age strata in this paper if adherence was separated in the future 
 
 
-.. list-table:: Affected Outcome #1 Effect Size
-  :widths: 15 15 15 
+This intervention affects primary adherence to medications, which in turn will affect the simulants exposure to SBP and LDL-C. 
+
+
+This odds ratio above will approximately halve the number of simulants that are primary non-adherent. All simulants that 
+are no longer primary non-adherent, are assumed to be adherent. This means that the new adherence rates for simulants 
+with the intervention are: 
```

Comment:
I think it would be useful to provide a bit more context here regarding the update (date that the change was made and maybe a link to the PR where the changes were made so that we can have a record of it). There's an example of this kind of note at the top of the SQ-LNS intervention page https://vivarium-research.readthedocs.io/en/latest/intervention_models/lipid_based_nutrient_supplements/index.html

Maybe it's not as important for this application though - so really whatever you think is best!